### ISSUE TITLE
Connection failure on Fedora Silverblue 38

### ISSUE THREAD

**Issue by sith-on-mars** [Windscribe/Desktop-App#88]
I couldn't connect to Silverblue no matter which connection method I tried. Windscribe worked on OpenSUSE MicroOS so I don't think it's just the general immutable system issue. Also, iVPN worked on my Silverblue.

Here is the output from the time I opened the app until the point I failed to make Wireguard connection and closed the app.

```
[dadstick@fedora ~]$ '/opt/windscribe/Windscribe' 
[{gmt_time}      0.025] [basic]	 App start time: "Wed Apr 26 17:06:02 2023"
[{gmt_time}      0.025] [basic]	 App version: "v2.6.14"
[{gmt_time}      0.025] [basic]	 Platform: "wayland"
[{gmt_time}      0.025] [basic]	 OS Version: "Linux #1 SMP PREEMPT_DYNAMIC Thu Apr 20 23:05:25 UTC 2023"
[{gmt_time}      0.025] [basic]	 CPU architecture: "x86_64"
[{gmt_time}      0.030] [basic]	 Detected OpenVPN versions: QList("2.5.4")
[{gmt_time}      0.031] [basic]	 Selected OpenVPN version: "2.5.4"
[{gmt_time}      0.031] [basic]	 DpiScaleManager::constructor -> DPI: 96 ; scale: 1 ; devicePixelRatio: 2
[{gmt_time}      0.034] [basic]	 OS in dark mode:  true
[{gmt_time}      0.043] [qt.core.qobject.connect]	 QObject::connect: No such signal QPlatformNativeInterface::systemTrayWindowChanged(QScreen*)
[{gmt_time}      0.044] [basic]	 GUI pid:  5130
[{gmt_time}      0.044] [basic]	 Gui settings {isLaunchOnStartup:true; isAutoConnect:false; isHideFromDock:false; isShowNotifications:false; orderLocation:"Geography"; latencyDisplay:"Bars"; shareSecureHotspot:{isEnabled:false; ssid:empty; password:empty}; shareProxyGateway:{isEnabled:false; proxySharingMode:"HTTP"}; splitTunneling:{settings:{active:false; mode:"Exclude"}; apps:QList(); networkRoutes:QList()}; isDockedToTray:false; isMinimizeAndCloseToTray:false; backgroundSettings:{backgroundType:1; backgroundImageDisconnected:empty; backgroundImageConnected:empty}; isStartMinimized:false; isShowLocationHealth:false; isAutoSecureNetworks:true; appSkin:"Alpha"}
[{gmt_time}      0.045] [basic]	 Engine settings {language:"en"; updateChannel:"Release"; isIgnoreSslErrors:false; isTerminateSockets:true; isAllowLanTraffic:false; firewallSettings: {mode:"Automatic"; when:"Before Connection"}; connectionSettings: {isAutomatic:true; protocol:"WireGuard"; port:443}; dnsResolutionSettings: {isAutomatic:true; manualAddress:""}; proxySettings: {option:"None"; address:""; port:0; username:empty; password:empty}; packetSize: {isAutomatic:true; mtu:-1}; macAddrSpoofing: {isEnabled:false; macAddress:""; selectedNetworkInterface:{friendlyName:""; networkOrSsid:""}; networkInterfaces:{}; dnsPolicy: "Cloudflare"; customOvpnConfigsPath: empty; isKeepAliveEnabled:false; connectedDnsInfo:{type:0}; dnsManager:"Automatic"; networkPreferredProtocols:QMap(); networkLastKnownGoodProtocols: {QMap(); }}
[{gmt_time}      1.400] [basic]	 Backend::init()
[{gmt_time}      1.401] [basic]	 connected to helper socket
[{gmt_time}      1.407] [basic]	 setWindowPosFromPersistent() - screen "Virtual-1" - geometry QRect(0,0 3200x1642) - virtualGeometry QRect(0,0 3200x1642) - logicalDotsPerInch 96 - devicePixelRatio 2
[{gmt_time}      1.407] [basic]	 setWindowPosFromPersistent() - restored app geometry: QRect(1404,633 392x376)
[{gmt_time}      1.407] [basic]	 Tray Icon geometry: QRect(0,0 0x0)
[{gmt_time}      1.411] [basic]	 OpenVPN helper connected ok
[{gmt_time}      1.416] [basic]	 DpiScaleManager::setMainWindow -> no DPI or pixel ratio changes
[{gmt_time}      1.447] [basic]	 Changed DNS servers for DnsResolver to: QList("1.1.1.1", "1.0.0.1")
[{gmt_time}      1.472] [network]	 SSL version: "OpenSSL 1.1.1t  7 Feb 2023"
[{gmt_time}      1.475] [basic]	 Curl version: libcurl/7.87.0 OpenSSL/3.1.0 zlib/1.2.13
[{gmt_time}      1.494] [basic]	 dga version: 2
[{gmt_time}      1.494] [server_api]	 ServerAPI::setApiResolutionsSettings {isAutomatic:true; manualAddress:""}
[{gmt_time}      1.495] [basic]	 "/usr/lib/opt/windscribe/windscribestunnel"
[{gmt_time}      1.495] [basic]	 "/usr/lib/opt/windscribe/windscribewstunnel"
[{gmt_time}      1.495] [best_location]	 Best location loaded from settings:  "651Atlanta - Piedmont"
[{gmt_time}      1.586] [basic]	 ImageResourcesSvg::run() - all SVGs loaded
[{gmt_time}      1.650] [basic]	 Firewall state from last app start: false
[{gmt_time}      1.661] [firewall_controller]	 firewall off
[{gmt_time}      1.674] [cli_ipc]	 IPC server for CLI started
[{gmt_time}      1.683] [failover]	 Trying: "acc: 139"
[{gmt_time}      1.688] [basic]	 Servers locations changed
[{gmt_time}      1.791] [basic]	 Network Changed:  Index:  2 , Network/SSID:  "enp1s0" , MAC:  "52:54:00:2A:B9:4F" , device name:  ""  friendly:  "enp1s0"
[{gmt_time}      1.898] [best_location]	 prevBestLocationId= "651Atlanta - Piedmont" ; prevBestLocationLatency= 8
[{gmt_time}      1.899] [best_location]	 Detected min latency= 6 ; id= "101Manchester - United"
[{gmt_time}      1.899] [best_location]	 Best location changed to  "101Manchester - United"
[{gmt_time}      3.084] [server_api]	 API request ApiAccessIps successfully executed
[{gmt_time}      3.469] [basic]	 Automatic enable firewall before connection
[{gmt_time}      3.469] [firewall_controller]	 firewall enabled with ips count: 181
[{gmt_time}      3.490] [basic]	 MutableLocationInfo created:  "Los Angeles - Lamar" "89.187.185.33; 185.152.67.225; 89.187.187.97; " ; Selected node: 2
[{gmt_time}      3.490] [basic]	 radiusUsername openvpn:  "rjznpbe8-z526vyc"
[{gmt_time}      3.490] [basic]	 radiusUsername ikev2:  "rjznpbe8-63te5b8"
[{gmt_time}      3.490] [basic]	 Connecting to "Los Angeles - Lamar"
[{gmt_time}      3.490] [connection]	 Connection settings: automatic
[{gmt_time}      3.490] [connection]	 "Location nodes: node1 = {ip1 = 89.187.185.33, ip2 = 89.187.185.34, ip3 = 89.187.185.35}; node2 = {ip1 = 185.152.67.225, ip2 = 185.152.67.226, ip3 = 185.152.67.227}; node3 = {ip1 = 89.187.187.97, ip2 = 89.187.187.98, ip3 = 89.187.187.99}; "
[{gmt_time}      3.490] [connection]	 Default adapter and gateway: "adapter name = enp1s0; adapter IP = 10.0.2.15; gateway IP = 10.0.2.2; remote IP = ; dns = (); ifIndex = not detected"
[{gmt_time}      3.490] [connection]	 Connecting to IP: "89.187.187.99"  protocol: "WireGuard"  port: 443
[{gmt_time}      3.490] [connection]	 Requesting WireGuard config for hostname = "us-west-066.whiskergalaxy.com"
[{gmt_time}      4.687] [basic]	 update session status (changed since last call)
[{gmt_time}      4.687] [basic]	 "[SessionStatus] { is_premium: 1; status: 1; rebill: 0; billing_plan_id: 108; premium_expire_date: 2026-03-20; traffic_used: 322264579653; traffic_max: -1; email_status: 0; static_ips: 0; alc_count: 0; last_reset_date: 2023-03-29 }"
[{gmt_time}      5.927] [basic]	 Received Check Update Answer
[{gmt_time}      5.928] [basic]	 No available update
[{gmt_time}      5.930] [server_api]	 API request ServerConfigs successfully executed
[{gmt_time}      6.005] [server_api]	 API request ServerCredentials "openvpn" successfully executed
[{gmt_time}      6.059] [server_api]	 WgConfigs connect request successfully executed
[{gmt_time}      6.059] [basic]	 Whitelist connecting ip: "89.187.187.99"
[{gmt_time}      6.059] [basic]	 Whitelist DNS-server ip: "10.255.255.3"
[{gmt_time}      6.067] [firewall_controller]	 firewall enabled with ips count: 183
[{gmt_time}      6.103] [connection]	 Connecting WireGuard: "utun420"
[{gmt_time}      6.109] [server_api]	 Notifications request successfully executed
[{gmt_time}      6.109] [wireguard]	 WireGuard started after 0 retries
[{gmt_time}      6.109] [server_api]	 API request PortMap successfully executed
[{gmt_time}      6.109] [wireguard]	 Configuring WireGuard...
[{gmt_time}      6.163] [basic]	 DNS-manager configuration: isResolvConfInstalled = true ; resolvConfSymlink = "/usr/bin/resolvectl" ; isSystemdResolvedServiceRunning = true ; resolvConfFileSymlink = "/run/systemd/resolve/stub-resolv.conf"
[{gmt_time}      6.164] [basic]	 /etc/resolv.conf header: "# This is /run/systemd/resolve/stub-resolv.conf managed by man:systemd-resolved(8).# Do not edit.## This file might be symlinked as /etc/resolv.conf. If you're looking at# /etc/resolv.conf and seeing this text, you have followed the symlink.## This is a dynamic resolv.conf file for connecting local clients to the# internal DNS stub resolver of systemd-resolved. This file lists all# configured search domains.## Run \"resolvectl status\" to see details about the uplink DNS servers# currently in use.## Third party programs should typically not access this file directly, but only# through the symlink at /etc/resolv.conf. To manage man:resolv.conf(5) in a# different way, replace this symlink by a static file or a different symlink.## See man:systemd-resolved.service(8) for details about the supported modes of# operation for /etc/resolv.conf."
[{gmt_time}      6.164] [basic]	 The DNS installation method -> resolvconf
[{gmt_time}      6.168] [server_api]	 API request ServerCredentials "ikev2" successfully executed
[{gmt_time}      6.243] [wireguard]	 WireGuard configuration failed
[{gmt_time}      6.243] [basic]	 Disconnected from helper socket, try reconnect
[{gmt_time}      6.244] [wireguard]	 Failed to configure WireGuard
[{gmt_time}      6.244] [connection]	 ConnectionManager::onConnectionError(), state_ = 1 , error = 32
[{gmt_time}      6.244] [basic]	 on reconnecting event
[{gmt_time}      6.244] [basic]	 Changed DNS servers for DnsResolver to: QList("1.1.1.1", "1.0.0.1")
[{gmt_time}      6.244] [basic]	 FirewallController_linux::setInterfaceToSkip_posix -> "utun420"
[{gmt_time}      6.244] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 3
[{gmt_time}      6.244] [basic]	 connected to helper socket
[{gmt_time}      6.245] [connection]	 Default adapter and gateway: "adapter name = enp1s0; adapter IP = 10.0.2.15; gateway IP = 10.0.2.2; remote IP = ; dns = (); ifIndex = not detected"
[{gmt_time}      6.245] [connection]	 Connecting to IP: "89.187.187.99"  protocol: "WireGuard"  port: 443
[{gmt_time}      6.245] [connection]	 Requesting WireGuard config for hostname = "us-west-066.whiskergalaxy.com"
[{gmt_time}      6.412] [wireguard]	 WireGuard daemon stopped after 0 retries
[{gmt_time}      6.413] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 3
[{gmt_time}      6.413] [connection]	 Default adapter and gateway: "adapter name = enp1s0; adapter IP = 10.0.2.15; gateway IP = 10.0.2.2; remote IP = ; dns = (); ifIndex = not detected"
[{gmt_time}      6.413] [connection]	 Connecting to IP: "89.187.187.99"  protocol: "WireGuard"  port: 443
[{gmt_time}      6.413] [connection]	 Requesting WireGuard config for hostname = "us-west-066.whiskergalaxy.com"
[{gmt_time}      6.476] [server_api]	 API request ServerLocations successfully executed, revision changed = 54054 , revision_hash = "2f06137269c154e1b70327fa29e0283603618f30"
[{gmt_time}      6.526] [basic]	 Servers locations changed
[{gmt_time}      6.535] [firewall_controller]	 firewall enabled with ips count: 183
[{gmt_time}      7.488] [server_api]	 WgConfigs connect request successfully executed
[{gmt_time}      7.489] [basic]	 Whitelist connecting ip: "89.187.187.99"
[{gmt_time}      7.489] [basic]	 Whitelist DNS-server ip: "10.255.255.3"
[{gmt_time}      7.489] [connection]	 Connecting WireGuard: "utun420"
[{gmt_time}      7.502] [wireguard]	 WireGuard started after 0 retries
[{gmt_time}      7.502] [wireguard]	 Configuring WireGuard...
[{gmt_time}      7.562] [basic]	 DNS-manager configuration: isResolvConfInstalled = true ; resolvConfSymlink = "/usr/bin/resolvectl" ; isSystemdResolvedServiceRunning = true ; resolvConfFileSymlink = "/run/systemd/resolve/stub-resolv.conf"
[{gmt_time}      7.562] [basic]	 /etc/resolv.conf header: "# This is /run/systemd/resolve/stub-resolv.conf managed by man:systemd-resolved(8).# Do not edit.## This file might be symlinked as /etc/resolv.conf. If you're looking at# /etc/resolv.conf and seeing this text, you have followed the symlink.## This is a dynamic resolv.conf file for connecting local clients to the# internal DNS stub resolver of systemd-resolved. This file lists all# configured search domains.## Run \"resolvectl status\" to see details about the uplink DNS servers# currently in use.## Third party programs should typically not access this file directly, but only# through the symlink at /etc/resolv.conf. To manage man:resolv.conf(5) in a# different way, replace this symlink by a static file or a different symlink.## See man:systemd-resolved.service(8) for details about the supported modes of# operation for /etc/resolv.conf."
[{gmt_time}      7.562] [basic]	 The DNS installation method -> resolvconf
[{gmt_time}      7.630] [wireguard]	 WireGuard configuration failed
[{gmt_time}      7.631] [basic]	 Disconnected from helper socket, try reconnect
[{gmt_time}      7.631] [wireguard]	 Failed to configure WireGuard
[{gmt_time}      7.631] [connection]	 ConnectionManager::onConnectionError(), state_ = 3 , error = 32
[{gmt_time}      7.631] [basic]	 connected to helper socket
[{gmt_time}      7.632] [basic]	 FirewallController_linux::setInterfaceToSkip_posix -> "utun420"
[{gmt_time}      7.716] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 3
[{gmt_time}      7.733] [wireguard]	 WireGuard daemon stopped after 0 retries
[{gmt_time}      7.734] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 3
[{gmt_time}     11.605] [connection]	 ConnectionManager::clickDisconnect()
[{gmt_time}     11.605] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 4
[{gmt_time}     11.605] [basic]	 on disconnected event
[{gmt_time}     11.605] [basic]	 Changed DNS servers for DnsResolver to: QList("1.1.1.1", "1.0.0.1")
[{gmt_time}     11.605] [basic]	 FirewallController_linux::setInterfaceToSkip_posix -> ""
[{gmt_time}     11.616] [firewall_controller]	 firewall enabled with ips count: 181
[{gmt_time}     11.666] [firewall_controller]	 firewall off
[{gmt_time}     14.360] [basic]	 Backend::cleanup()
[{gmt_time}     14.360] [basic]	 Firewall on next startup:  false
[{gmt_time}     14.360] [basic]	 Cleanup started
[{gmt_time}     14.360] [basic]	 Cleanup, connection manager no need disconnect
[{gmt_time}     14.428] [basic]	 close main window
[{gmt_time}     14.489] [basic]	 Cleanup finished
[{gmt_time}     14.489] [basic]	 Backend Cleanup Finished
[{gmt_time}     14.507] [ipc]	 IPC server stopped
[{gmt_time}     14.525] [cli_ipc]	 IPC server for CLI stopped
warning: queue 0x2bccc90 destroyed while proxies still attached:
  wl_display@1 still attached
[{gmt_time}     14.535] [basic]	 Stopping DnsResolver
[{gmt_time}     14.535] [basic]	 DnsResolver stopped
```

**Comment by bernerdad** [comment]
Can you provide us with the contents of `/opt/windscribe/helper_log.txt`?

**Comment by sith-on-mars** [comment]
I didn't find `helper_log.txt` you're referring to. I found the debug log under setting => help, which I hope it's helpful.

```
G [270423 01:48:46:199      0.017] [basic]	 App start time: "Thu Apr 27 11:48:46 2023"
G [270423 01:48:46:199      0.017] [basic]	 App version: "v2.6.14"
G [270423 01:48:46:199      0.017] [basic]	 Platform: "wayland"
G [270423 01:48:46:199      0.017] [basic]	 OS Version: "Linux #1 SMP PREEMPT_DYNAMIC Thu Apr 20 23:05:25 UTC 2023"
G [270423 01:48:46:199      0.017] [basic]	 CPU architecture: "x86_64"
G [270423 01:48:46:203      0.021] [basic]	 Detected OpenVPN versions: QList("2.5.4")
G [270423 01:48:46:203      0.021] [basic]	 Selected OpenVPN version: "2.5.4"
G [270423 01:48:46:204      0.021] [basic]	 DpiScaleManager::constructor -> DPI: 96 ; scale: 1 ; devicePixelRatio: 2
G [270423 01:48:46:208      0.026] [basic]	 OS in dark mode:  true
G [270423 01:48:46:222      0.040] [basic]	 GUI pid:  25922
G [270423 01:48:46:223      0.041] [basic]	 Gui settings {isLaunchOnStartup:false; isAutoConnect:true; isHideFromDock:false; isShowNotifications:false; orderLocation:"Alphabet"; latencyDisplay:"Ms"; shareSecureHotspot:{isEnabled:false; ssid:empty; password:empty}; shareProxyGateway:{isEnabled:false; proxySharingMode:"HTTP"}; splitTunneling:{settings:{active:false; mode:"Exclude"}; apps:QList(); networkRoutes:QList()}; isDockedToTray:false; isMinimizeAndCloseToTray:true; backgroundSettings:{backgroundType:1; backgroundImageDisconnected:empty; backgroundImageConnected:empty}; isStartMinimized:true; isShowLocationHealth:true; isAutoSecureNetworks:true; appSkin:"Alpha"}
G [270423 01:48:46:224      0.042] [basic]	 Engine settings {language:"en"; updateChannel:"Release"; isIgnoreSslErrors:false; isTerminateSockets:true; isAllowLanTraffic:true; firewallSettings: {mode:"Automatic"; when:"Before Connection"}; connectionSettings: {isAutomatic:true; protocol:"WireGuard"; port:443}; dnsResolutionSettings: {isAutomatic:true; manualAddress:""}; proxySettings: {option:"None"; address:""; port:0; username:empty; password:empty}; packetSize: {isAutomatic:true; mtu:-1}; macAddrSpoofing: {isEnabled:false; macAddress:""; selectedNetworkInterface:{friendlyName:""; networkOrSsid:""}; networkInterfaces:{}; dnsPolicy: "Cloudflare"; customOvpnConfigsPath: empty; isKeepAliveEnabled:false; connectedDnsInfo:{type:0}; dnsManager:"Automatic"; networkPreferredProtocols:QMap(); networkLastKnownGoodProtocols: {QMap(); }}
G [270423 01:48:49:679      3.496] [basic]	 Backend::init()
G [270423 01:48:49:679      3.497] [basic]	 connected to helper socket
G [270423 01:48:49:686      3.504] [basic]	 setWindowPosFromPersistent() - screen "eDP-1" - geometry QRect(0,0 3200x1800) - virtualGeometry QRect(0,0 3200x1800) - logicalDotsPerInch 96 - devicePixelRatio 2
G [270423 01:48:49:686      3.504] [basic]	 setWindowPosFromPersistent() - restored app geometry: QRect(1361,823 392x376)
G [270423 01:48:49:686      3.504] [basic]	 Tray Icon geometry: QRect(0,0 0x0)
G [270423 01:48:49:689      3.507] [basic]	 OpenVPN helper connected ok
G [270423 01:48:49:696      3.513] [basic]	 DpiScaleManager::setMainWindow -> no DPI or pixel ratio changes
G [270423 01:48:49:713      3.531] [basic]	 Changed DNS servers for DnsResolver to: QList("1.1.1.1", "1.0.0.1")
G [270423 01:48:49:721      3.539] [network]	 SSL version: "OpenSSL 1.1.1t  7 Feb 2023"
G [270423 01:48:49:722      3.539] [basic]	 Curl version: libcurl/7.87.0 OpenSSL/3.1.0 zlib/1.2.13
G [270423 01:48:49:734      3.552] [basic]	 dga version: 2
G [270423 01:48:49:734      3.552] [server_api]	 ServerAPI::setApiResolutionsSettings {isAutomatic:true; manualAddress:""}
G [270423 01:48:49:734      3.552] [basic]	 "/usr/lib/opt/windscribe/windscribestunnel"
G [270423 01:48:49:734      3.552] [basic]	 "/usr/lib/opt/windscribe/windscribewstunnel"
G [270423 01:48:49:735      3.552] [best_location]	 Best location loaded from settings:  "[redacted]"
G [270423 01:48:49:803      3.621] [basic]	 ImageResourcesSvg::run() - all SVGs loaded
G [270423 01:48:49:845      3.663] [basic]	 Firewall state from last app start: false
G [270423 01:48:49:851      3.668] [firewall_controller]	 firewall off
G [270423 01:48:49:856      3.674] [cli_ipc]	 IPC server for CLI started
G [270423 01:48:49:863      3.680] [failover]	 Trying: "acc: 139"
G [270423 01:48:49:866      3.684] [basic]	 Servers locations changed
G [270423 01:48:49:918      3.736] [basic]	 Network Whitelisting detected SECURED network -- Connecting..
G [270423 01:48:49:928      3.746] [basic]	 Network Changed:  Index:  3 , Network/SSID:  "ANU-Secure" , MAC:  "64:5D:86:D7:52:17" , device name:  ""  friendly:  "ANU-Secure"
G [270423 01:48:49:932      3.750] [basic]	 Automatic enable firewall before connection
G [270423 01:48:49:933      3.750] [firewall_controller]	 firewall enabled with ips count: 180
G [270423 01:48:49:950      3.768] [basic]	 MutableLocationInfo created:  "Los Angeles - Lamar" "185.152.67.225; 89.187.187.97; 89.187.185.33; " ; Selected node: 1
G [270423 01:48:49:950      3.768] [basic]	 radiusUsername openvpn:  "rjznpbe8-z526vyc"
G [270423 01:48:49:950      3.768] [basic]	 radiusUsername ikev2:  "rjznpbe8-63te5b8"
G [270423 01:48:49:950      3.768] [basic]	 Connecting to "Los Angeles - Lamar"
G [270423 01:48:49:950      3.768] [connection]	 Connection settings: automatic
G [270423 01:48:49:950      3.768] [connection]	 "Location nodes: node1 = {ip1 = 185.152.67.225, ip2 = 185.152.67.226, ip3 = 185.152.67.227}; node2 = {ip1 = 89.187.187.97, ip2 = 89.187.187.98, ip3 = 89.187.187.99}; node3 = {ip1 = 89.187.185.33, ip2 = 89.187.185.34, ip3 = 89.187.185.35}; "
G [270423 01:48:49:951      3.768] [connection]	 Default adapter and gateway: "adapter name = wlp0s20f3; adapter IP = 10.28.104.177; gateway IP = 10.28.96.1; remote IP = ; dns = (); ifIndex = not detected"
G [270423 01:48:49:951      3.768] [connection]	 Connecting to IP: "89.187.187.99"  protocol: "WireGuard"  port: 443
G [270423 01:48:49:951      3.768] [connection]	 Requesting WireGuard config for hostname = "us-west-066.whiskergalaxy.com"
G [270423 01:48:51:971      5.789] [server_api]	 API request ApiAccessIps successfully executed
G [270423 01:48:51:994      5.812] [firewall_controller]	 firewall enabled with ips count: 181
G [270423 01:48:54:168      7.986] [basic]	 update session status (changed since last call)
G [270423 01:48:54:168      7.986] [basic]	 "[SessionStatus] { is_premium: 1; status: 1; rebill: 0; billing_plan_id: 108; premium_expire_date: 2026-03-20; traffic_used: 322631953514; traffic_max: -1; email_status: 0; static_ips: 0; alc_count: 0; last_reset_date: 2023-03-29 }"
G [270423 01:48:56:358     10.176] [server_api]	 API request ServerCredentials "openvpn" successfully executed
G [270423 01:48:56:393     10.211] [server_api]	 API request ServerConfigs successfully executed
G [270423 01:48:56:422     10.240] [server_api]	 Notifications request successfully executed
G [270423 01:48:56:423     10.241] [server_api]	 API request PortMap successfully executed
G [270423 01:48:56:451     10.268] [server_api]	 WgConfigs connect request successfully executed
G [270423 01:48:56:451     10.269] [basic]	 Whitelist connecting ip: "89.187.187.99"
G [270423 01:48:56:451     10.269] [basic]	 Whitelist DNS-server ip: "10.255.255.3"
G [270423 01:48:56:456     10.274] [firewall_controller]	 firewall enabled with ips count: 183
G [270423 01:48:56:474     10.292] [connection]	 Connecting WireGuard: "utun420"
G [270423 01:48:56:478     10.296] [server_api]	 API request ServerCredentials "ikev2" successfully executed
G [270423 01:48:56:478     10.296] [wireguard]	 WireGuard started after 0 retries
G [270423 01:48:56:478     10.296] [wireguard]	 Configuring WireGuard...
G [270423 01:48:56:508     10.326] [basic]	 DNS-manager configuration: isResolvConfInstalled = true ; resolvConfSymlink = "/usr/bin/resolvectl" ; isSystemdResolvedServiceRunning = true ; resolvConfFileSymlink = "/run/systemd/resolve/stub-resolv.conf"
G [270423 01:48:56:508     10.326] [basic]	 /etc/resolv.conf header: "# This is /run/systemd/resolve/stub-resolv.conf managed by man:systemd-resolved(8).# Do not edit.## This file might be symlinked as /etc/resolv.conf. If you're looking at# /etc/resolv.conf and seeing this text, you have followed the symlink.## This is a dynamic resolv.conf file for connecting local clients to the# internal DNS stub resolver of systemd-resolved. This file lists all# configured search domains.## Run \"resolvectl status\" to see details about the uplink DNS servers# currently in use.## Third party programs should typically not access this file directly, but only# through the symlink at /etc/resolv.conf. To manage man:resolv.conf(5) in a# different way, replace this symlink by a static file or a different symlink.## See man:systemd-resolved.service(8) for details about the supported modes of# operation for /etc/resolv.conf."
G [270423 01:48:56:508     10.326] [basic]	 The DNS installation method -> resolvconf
G [270423 01:48:56:513     10.331] [basic]	 Received Check Update Answer
G [270423 01:48:56:513     10.331] [basic]	 No available update
G [270423 01:48:56:582     10.400] [wireguard]	 WireGuard configuration failed
G [270423 01:48:56:583     10.400] [basic]	 Disconnected from helper socket, try reconnect
G [270423 01:48:56:583     10.401] [wireguard]	 Failed to configure WireGuard
G [270423 01:48:56:583     10.401] [connection]	 ConnectionManager::onConnectionError(), state_ = 1 , error = 32
G [270423 01:48:56:583     10.401] [basic]	 on reconnecting event
G [270423 01:48:56:583     10.401] [basic]	 Changed DNS servers for DnsResolver to: QList("1.1.1.1", "1.0.0.1")
G [270423 01:48:56:583     10.401] [basic]	 FirewallController_linux::setInterfaceToSkip_posix -> "utun420"
G [270423 01:48:56:583     10.401] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 3
G [270423 01:48:56:583     10.401] [basic]	 connected to helper socket
G [270423 01:48:56:584     10.402] [connection]	 Default adapter and gateway: "adapter name = wlp0s20f3; adapter IP = 10.28.104.177; gateway IP = 10.28.96.1; remote IP = ; dns = (); ifIndex = not detected"
G [270423 01:48:56:584     10.402] [connection]	 Connecting to IP: "89.187.187.99"  protocol: "WireGuard"  port: 443
G [270423 01:48:56:584     10.402] [connection]	 Requesting WireGuard config for hostname = "us-west-066.whiskergalaxy.com"
G [270423 01:48:56:747     10.565] [wireguard]	 WireGuard daemon stopped after 0 retries
G [270423 01:48:56:747     10.565] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 3
G [270423 01:48:56:748     10.565] [connection]	 Default adapter and gateway: "adapter name = wlp0s20f3; adapter IP = 10.28.104.177; gateway IP = 10.28.96.1; remote IP = ; dns = (); ifIndex = not detected"
G [270423 01:48:56:748     10.565] [connection]	 Connecting to IP: "89.187.187.99"  protocol: "WireGuard"  port: 443
G [270423 01:48:56:748     10.565] [connection]	 Requesting WireGuard config for hostname = "us-west-066.whiskergalaxy.com"
G [270423 01:48:56:979     10.796] [server_api]	 API request ServerLocations successfully executed, revision changed = 54087 , revision_hash = "e960f1daedc2d32b52d395483b198cebba219cab"
G [270423 01:48:57:019     10.837] [basic]	 Servers locations changed
G [270423 01:48:57:030     10.848] [firewall_controller]	 firewall enabled with ips count: 183
G [270423 01:48:58:229     12.047] [server_api]	 WgConfigs connect request successfully executed
G [270423 01:48:58:230     12.048] [basic]	 Whitelist connecting ip: "89.187.187.99"
G [270423 01:48:58:230     12.048] [basic]	 Whitelist DNS-server ip: "10.255.255.3"
G [270423 01:48:58:230     12.048] [connection]	 Connecting WireGuard: "utun420"
G [270423 01:48:58:240     12.058] [wireguard]	 WireGuard started after 0 retries
G [270423 01:48:58:240     12.058] [wireguard]	 Configuring WireGuard...
G [270423 01:48:58:281     12.099] [basic]	 DNS-manager configuration: isResolvConfInstalled = true ; resolvConfSymlink = "/usr/bin/resolvectl" ; isSystemdResolvedServiceRunning = true ; resolvConfFileSymlink = "/run/systemd/resolve/stub-resolv.conf"
G [270423 01:48:58:281     12.099] [basic]	 /etc/resolv.conf header: "# This is /run/systemd/resolve/stub-resolv.conf managed by man:systemd-resolved(8).# Do not edit.## This file might be symlinked as /etc/resolv.conf. If you're looking at# /etc/resolv.conf and seeing this text, you have followed the symlink.## This is a dynamic resolv.conf file for connecting local clients to the# internal DNS stub resolver of systemd-resolved. This file lists all# configured search domains.## Run \"resolvectl status\" to see details about the uplink DNS servers# currently in use.## Third party programs should typically not access this file directly, but only# through the symlink at /etc/resolv.conf. To manage man:resolv.conf(5) in a# different way, replace this symlink by a static file or a different symlink.## See man:systemd-resolved.service(8) for details about the supported modes of# operation for /etc/resolv.conf."
G [270423 01:48:58:281     12.099] [basic]	 The DNS installation method -> resolvconf
G [270423 01:48:58:332     12.150] [wireguard]	 WireGuard configuration failed
G [270423 01:48:58:332     12.150] [basic]	 Disconnected from helper socket, try reconnect
G [270423 01:48:58:332     12.150] [wireguard]	 Failed to configure WireGuard
G [270423 01:48:58:333     12.150] [connection]	 ConnectionManager::onConnectionError(), state_ = 3 , error = 32
G [270423 01:48:58:333     12.150] [basic]	 connected to helper socket
G [270423 01:48:58:333     12.150] [basic]	 FirewallController_linux::setInterfaceToSkip_posix -> "utun420"
G [270423 01:48:58:404     12.222] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 3
G [270423 01:48:58:433     12.251] [wireguard]	 WireGuard daemon stopped after 0 retries
G [270423 01:48:58:433     12.251] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 3
G [270423 01:49:04:664     18.482] [connection]	 ConnectionManager::clickDisconnect()
G [270423 01:49:04:664     18.482] [connection]	 ConnectionManager::onConnectionDisconnected(), state_ = 4
G [270423 01:49:04:664     18.482] [basic]	 on disconnected event
G [270423 01:49:04:664     18.482] [basic]	 Changed DNS servers for DnsResolver to: QList("1.1.1.1", "1.0.0.1")
G [270423 01:49:04:664     18.482] [basic]	 FirewallController_linux::setInterfaceToSkip_posix -> ""
G [270423 01:49:04:675     18.492] [firewall_controller]	 firewall enabled with ips count: 181
G [270423 01:49:04:729     18.546] [firewall_controller]	 firewall off
G [270423 01:49:10:061     23.879] [best_location]	 prevBestLocationId= "[redacted]" ; prevBestLocationLatency= 15
G [270423 01:49:10:062     23.879] [best_location]	 Detected min latency= 7 ; id= "[redacted]"
G [270423 01:49:10:062     23.879] [best_location]	 Best location changed to  "[redacted]"
```

**Comment by bernerdad** [comment]
Hmm... something must be happening with the helper, as it should be logging to that file.  As silverblue is not a currently supported distro, it may be quite some time before we have resources available to investigate this further.  Sorry!

**Comment by bernerdad** [comment]
@sith-on-mars If you're still using this OS, could you let us know if the latest guinea pig [build](https://github.com/Windscribe/Desktop-App/releases/tag/v2.11.6) resolves the issue.  Thank you!

**Comment by np22-jpg** [comment]
> @sith-on-mars If you're still using this OS, could you let us know if the latest guinea pig [build](https://github.com/Windscribe/Desktop-App/releases/tag/v2.11.6) resolves the issue. Thank you!

I'm not @sith-on-mars, but I can confirm the build works when installed to the host with `rpm-ostree install windscribe_2.11.6_guinea_pig_x86_64_fedora.rpm`, and appears to connect, with no DNS leaks from what I can tell.

**Comment by sith-on-mars** [comment]
Thanks heaps. I was able to connect to VPN on the latest 2.11.6 build. However, the VPN logged out every time after reboot.

**Comment by jaxu** [comment]
Thanks for trying it out!  We are aware of the logout issue and will have that resolved before general release.

**Comment by sith-on-mars** [comment]
> Thanks heaps. I was able to connect to VPN on the latest 2.11.6 build. However, the VPN logged out every time after logging out from the system or reboot.

The VPN logout issue was resolved in the 2.11.7 version. Closing.
